### PR TITLE
[wip] Rebase crash pad add/remove extra

### PR DESCRIPTION
**Pull request by lepinay**
#### Description of Change

Please ignore this PR, just testing something out, will delete soon.

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes:

### CODE REVIEW

**Comment by nornagon** [comment]
Hi, I know this is still WIP, but can you please fill out the PR template? would love to know more about what you're aiming to accomplish :)

**Comment by lepinay** [comment]
@nornagon Sorry, I was just testing out something, I'll delete this soon

**Comment by miniak** [comment]
@lepinay if you have a PR for testing only, please create it as Draft and add the WIP label.

File: shell/app/node_main.cc [hunk 1]
```diff
@@ -84,6 +95,12 @@ int NodeMain(int argc, char* argv[]) {
     // Setup process.crashReporter.start in child node processes
     auto reporter = mate::Dictionary::CreateEmpty(gin_env.isolate());
     reporter.SetMethod("start", &crash_reporter::CrashReporter::StartInstance);
+
+#if defined(OS_MACOSX)
```
Comment (miniak) [inline shell/app/node_main.cc]: should be `#if !defined(OS_LINUX)` as well?

**Pull request closed without merging**